packages/story-editor/src/components/panels/design/textAccessibility/textAccessibility.js

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { __, sprintf, TranslateWithMarkup } from '@googleforcreators/i18n';
+import {
+  DropDown,
+  Text,
+  THEME_CONSTANTS,
+} from '@googleforcreators/design-system';
+import { getTextElementTagNames } from '@googleforcreators/output';
+import { trackClick } from '@googleforcreators/tracking';
+import { useFeature } from 'flagged';
+import { useState, useEffect } from '@googleforcreators/react';
+
+/**
+ * Internal dependencies
+ */
+import { Row } from '../../../form';
+import { getCommonValue } from '../../shared';
+import { SimplePanel } from '../../panel';
+import { MULTIPLE_VALUE, MULTIPLE_DISPLAY_VALUE } from '../../../../constants';
+import { combineElementsWithTags } from './utils';
+
+const HEADING_LEVELS = {
+  h1: __('Heading 1', 'web-stories'),
+  h2: __('Heading 2', 'web-stories'),
+  h3: __('Heading 3', 'web-stories'),
+  p: __('Paragraph', 'web-stories'),
+};
+
+function TextAccessibilityPanel({ selectedElements, pushUpdate }) {
+  // Feature flagging for Semantic Headings
+  const showSemanticHeadings = useFeature('showSemanticHeadings');
+  const [selectedTextElements, setSelectedTextElements] = useState([]);
+  const [currentTags, setCurrentTags] = useState([]);
+
+  useEffect(() => {
+    // We want to get the text tags for the elements if not already defined
+    const textTags = getTextElementTagNames(selectedElements);
+    // Then combine the elements with their associated tag
+    const newElements = combineElementsWithTags(selectedElements, textTags);
+    // And then set them into component state
+    setSelectedTextElements(newElements);
+    setCurrentTags(Array.from(textTags.values()));
+  }, [selectedElements, setSelectedTextElements]);
+
+  if (!showSemanticHeadings) {
+    return null;
+  }
+
+  // Map all types of tag names in the selected elements
+  // and then convert to an Array for usage
+  const handleChange = (evt, value) => {
+    trackClick(evt, 'click-headings-docs');
```

Comment:
As per my other comment, this should be removed. We want to track clicks on the "Learn more" link.

Reply:
Removed, thank you! https://github.com/GoogleForCreators/web-stories-wp/pull/11888/commits/705ece200af422cea010cd4b6b17fe5843cddee3